libtenzir/builtins/formats/lines.cpp

```diff
@@ -261,9 +276,19 @@ class read_lines final
     auto args = parser_args{};
     argument_parser2::operator_(name())
       .named("skip_empty", args.skip_empty)
+      .named("split_regex", args.split_regex)
       .named("split_at_null", args.null)
```

Comment:
We should be consistent with the naming here:

- `split_at_regex` and `split_at_null`, or
- `split_regex` and `slit_nulls`

Reply:
Fine by me, let's use `split_at_regex` to avoid a breaking change for the other option.